src/__tests__/exampleData.js

```diff
@@ -90,16 +90,29 @@ export const makeCrossRealmBot = (args: { name?: string } = {}): CrossRealmBot =
     is_bot: true,
   });
 
-const makeAccount = (user: User): Account =>
-  deepFreeze({
-    realm: 'https://zulip.example.org',
-    email: user.email,
-    apiKey: randString() + randString(),
+export const makeAccount = (args: {
+  email: string,
+  realm: string,
+  shouldHaveApiKey?: boolean,
+  shouldHaveZulipVersion?: boolean,
+}): Account => {
+  const { email, realm, shouldHaveApiKey = false, shouldHaveZulipVersion = false } = args;
```

Comment:
One thing that helps keep the code in the test cases themselves simple is when they can skip mentioning options for things that are boring for that test case, and only mention something like the API key when the point of the test case is specifically related to whether the API key is present or absent.

It looks like in the current version, this works out very nicely for `shouldHaveZulipVersion` -- it's actually not mentioned in any call sites except `selfAccount`.

For `shouldHaveApiKey`, all but a few of the call sites are passing `true`. That's a sign that that should be the default -- most of those test cases aren't really thinking about the API key, and it's just a background assumption of a lot of our code that there is an API key in the active account.